kubernetes/cmsweb/services/das-mongo-exporter.yaml

```diff
@@ -60,5 +60,5 @@ spec:
           name: das-mongo-p
       initContainers:
       - name: checkcouchdb
-        image: busybox:1.28
-        command: ['sh', '-c', 'until nslookup das-mongo.das; do echo "Waiting for das-mongo"; sleep 10; done;']
+        image: busybox:1.36.1 #the stable tag
```

Comment:
if you have  tested with the latest stable tag like: 
`busybox:stable` please change it to this one instead of fixed number

Reply:
the stable tag is not resolving. That's why I used this. I can upload this image to registry.

Reply:
ok